packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -362,6 +362,21 @@ export default class LocusInfo extends EventsScope {
           }
         );
       }
+      // If you are  guest and you are removed from the meeting
+      // You wont get any further events
```

Comment:
I thought that was the intended behavior? Once you're booted you shouldn't care about the meeting any more.

Reply:
There are space meeting scenario where you are in IDLE state but you will get mercury event . In this case the meeting is still active but he is removed so we nee to remove the meeting object